system/timed.py

```diff
@@ -41,13 +46,13 @@ def main() -> NoReturn:
     msg.clocks.wallTimeNanos = time.time_ns()
     pm.send('clocks', msg)
 
-    llk = sm['liveLocationKalman']
-    if not llk.gpsOK or (time.monotonic() - sm.logMonoTime['liveLocationKalman']/1e9) > 0.2:
+    gps = sm[gps_location_service]
+    if not sm.updated[gps_location_service] or (time.monotonic() - sm.logMonoTime[gps_location_service] / 1e9) > 0.2:
```

Comment:
This time threshold cannot be the same for both services, since they're being published at different frequencies (1hz vs 10hz)